flexmeasures/cli/data_show.py

```diff
@@ -376,4 +376,27 @@ def plot_beliefs(
         click.secho("Data saved to file.", **MsgStyle.SUCCESS)
 
 
+@fm_show_data.command("reporters")
+@with_appcontext
+def list_reporters():
+    """
+    Show available reporters.
+    """
+
+    click.echo("Reporters:\n")
+    click.echo(
+        tabulate(
+            [
+                (
+                    reporter_name,
+                    reporter_class.__version__,
+                    str(reporter_class.__author__),
```

Comment:
While imagining a long list of reporters, some with authors, some without, I actually prefer `None` values showing as empty entries under the author column.

Reply:
yes, that make sense.